script.module.future/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.future" name="future" version="0.16.0" provider-name="Ed Schofield">
+<addon id="script.module.future" name="future" version="0.16.0.1" provider-name="Ed Schofield">
```

Comment:
wasn't sure about that either.
Should I go for `0.16.0~1` than? I'm really clueless on how to give that thing a somewhat proper version number in this case :/
Or `0.16.0~restructured`? I hate version-number that have anything other than `.` and `0-9` :D